src/lib.rs

```diff
@@ -19,9 +19,47 @@ pub use tables::GeneralCategory;
 /// that this version of unicode-general-category was generated from.
 pub const UNICODE_VERSION: (u64, u64, u64) = (16, 0, 0);
 
+impl GeneralCategory {
+    /// Returns the general category abbreviation.
+    pub fn as_str(&self) -> &'static str {
```

Comment:
Since this does not return the general category as a string, instead its abbreviation, please use an alternate name like `abbreviation`, `abbreviated_name`, `short_name`, or similar.